core/src/main/java/com/redhat/lightblue/client/LightblueClientConfiguration.java

```diff
@@ -54,6 +59,33 @@ public LightblueClientConfiguration(LightblueClientConfiguration configuration)
         maxQueryTimeMS = configuration.maxQueryTimeMS;
     }
 
+    /**
+     *
+     * @return boolean whether or not the caFilePath value is actually a delimited list
+     * representing multiple CA certs
+     */
+    public boolean hasMultipleCaCerts() {
+        return (caFilePath.contains(CA_CERT_FILE_DELIMITER));
+    }
+
+    /**
+     *
+     * @return List<String> of caFilePaths generated from parsing the caFilePath
+     * value using the value specified by the CA_CERT_FILE_DELIMITER constant.
+     * If caFilePath value is not delimited, returns a List with only one path in it.
+     */
+    public List<String> getCaFilePaths() {
```

Comment:
How should getCaFilePath() behave when there are multiple CAs defined in the config file?